src/components/templates/Audience/Audience.tsx

```diff
@@ -461,7 +459,7 @@ export const Audience: React.FunctionComponent = () => {
                               seat && seatedPartygoer === null
                                 ? takeSeat(row, column)
                                 : seatedPartygoer !== null
-                                ? setSelectedUserProfile(seatedPartygoer)
+                                ? openUserProfileModal(seatedPartygoer)
                                 : null
                             }
```

Comment:
[NB, potentially OOS] I'm not 100%, but I feel like this is sort of legacy/leftover from ancient code, and may not even be required anymore. Theoretically, if I click on a seat that has a user in it, then the `UserProfilePicture` should be registering the click, and handling opening the modal for me. So I don't think we should also need to have explicit logic here to handle it?

We probably want to make sure we don't try and take the seat if there is already another user there still (which eventually should be something we handle/enforce on the backend probably, rather than just the frontend.

This whole section of the code is actually fairly legacy, as I know I refactored some almost exactly similar code that was used on the `PartyMap` to use `usePartygoersBySeat`/`usePartygoersOverlay` and related to handle this sort of stuff:

- https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/PartyMap/components/Map/Map.tsx
- https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/PartyMap/components/Map/hooks/usePartygoersBySeat.ts
- https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/PartyMap/components/Map/hooks/usePartygoersOverlay.tsx